app/views/export/_invitation.html.erb

```diff
@@ -0,0 +1,50 @@
+<tr>
+<td>
+  <%= invitation.invitee.try(:full_name) %> <%= invitation.email %>
+</td>
+<td>
+  <%if invitation.decision.major_version%>
+    v<%= invitation.decision.major_version %>.<%= invitation.decision.minor_version %>
+    <%= invitation.decision.verdict.try(:humanize) %>
+  <% end %>
+</td>
+<td>
+  <%if !invitation.pending? %>
+    <%if invitation.invited? %>
+      Invited <%= invitation.invited_at%>
+    <% else %>
+      <%if invitation.accepted? %>
+        <%if ReviewerReport.for_invitation(invitation) %>
```

Comment:
This method `for_invitation` doesn't actually do what you think it does. I remember this because it was a pain in the back. Bad data modeling. I'd consider omitting this, it wont correspond correctly to the invitation version number. It just gets the latest reviewer report, not the correct reviewer report for the corresponding decision version. https://github.com/Aperta-project/Aperta/blob/develop/app/models/reviewer_report.rb#L59-L66

Reply:
Wait, I take that back. There was a edge case where a user could have two reviewer reports for the same decision version, but probably doesn't matter for our exporter.